### ISSUE TITLE
Screen Reader: Many Pages: Informative dots are not announced

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77364]
## Action Performed:
Precondition: User is logged in

1. Using VoiceOver on iOS or TalkBack on Android, open the app
2. Swipe right to focus the Account tab and double-tap to activate it
3. Swipe right to focus the Profile button and double-tap to activate it
4. Swipe right to focus the Contact method menu item and note if the adjacent green dot is announced with it

Other Occurrences:
1. Profile > Contact methods: unverified email addresses with green dot
2. Account: menu items with green/red dot (e.g., Profile/Subscription)
3. Inbox: chats with green/red dot or pin icon

## Expected Result:
Screen readers should announce the informative dot icon’s meaning, e.g.:
Green icon - actions to take
Red icon - items to review

Example announcements:
"Contact method, menu item. There are actions to take" or more specifically "email verification required"
"Subscription, menu item. There are items to review" or more specifically "Payment processing failure"

## Actual Result:
The green/red dot is not communicated or exposed to screen readers users.
Users relying on screen readers cannot detect that an action is required.

## Area issue was found in:
Profile

## Failed WCAG checkpoints
WCAG 1.1.1 Non-Text Content
https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html

## User impact:
Users who have difficulty perceiving visual content use assistive technology to read text aloud, present it visually, or convert it to braille. Text alternatives may help some people who have difficulty understanding the meaning of photographs, drawings, and other images (e.g., line drawings, graphic designs, paintings, three-dimensional representations), graphs, charts, animations, etc. People who are deaf-blind can read the text alternatives in braille.

## Suggested resolution:
Integrate the meaning of the dot into the parent button’s accessibilityLabel to ensure it is communicated with it, for example:
"Contact method, menu item. There are actions to take" or more specifically "email verification required"
"Subscription, menu item. There are items to review" or more specifically "Payment processing failure"

Refer to:
https://appt.org/en/docs/react-native/samples/accessibility-label
https://reactnative.dev/docs/accessibility#accessibilitylabel

## Environment(s)
Apple iPhone 14 Pro - iOS 18.7.1, Wi-Fi - Native (No mobile browser)
Samsung Galaxy Note10 Plus - Android 12, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos
<img width="1179" height="2437" alt="Image" src="https://github.com/user-attachments/assets/5aab6f97-7ab1-413d-b205-5d48cff321b3" />

https://github.com/user-attachments/assets/f7eddee1-7887-498a-a5ea-10b71bdd612e

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7005957
**Issue reported by:** NA
**Slack conversation:** NA
